fairlearn/postprocessing/_interpolated_thresholder.py

```diff
@@ -57,6 +94,18 @@ def fit(self, X, y, **kwargs):
         if self.estimator is None:
             raise ValueError(BASE_ESTIMATOR_NONE_ERROR_MESSAGE)
 
+        if self.predict_method == "deprecated":
+            warn(
+                "'predict_method' default value is changed from 'predict' to "
+                "'auto'. Explicitly pass `predict_method='predict' to "
+                "replicate the old behavior, or pass `predict_method='auto' "
+                "or other valid values to silence this warning.",
+                FutureWarning,
+            )
+            self._predict_method = "predict"
+        else:
+            self._predict_method = self.predict_method
```

Comment:
These two properties feel like a way to turn typos into subtle bugs

Reply:
I'm assuming you're fine now with this solution since it won't be needed after the deprecation?

Reply:
Yes.... wouldn't want it long term, but if it's coming out in a release or two I'm fine with it.